pcdsdevices/sample_delivery.py

```diff
@@ -0,0 +1,435 @@
+
+    name : str
+        A name for the device.
+
+    integrator_source_prefix : str
+
+    flow_source_prefix : str
+
+    names_prefix : str
+        Names of.
+
+    start1_prefix : str
+        Starting volume of 1.
+
+    used1_prefix : str
+        Flow of 1.
+
+    time1_prefix : str
+        Estimated depletion time of 1.
+
+    start2_prefix : str
+        Starting volume of 2.
+
+    used2_prefix : str
+        Flow of 2.
+
+    time2_prefix : str
+        Estimated depletion time of 2.
+
+    start3_prefix : str
+        Starting volume of 3.
+
+    used3_prefix : str
+        Flow of 3.
+
+    time3_prefix : str
+        Estimated depletion time of 3.
+
+    start4_prefix : str
+        Starting volume of 4.
+
+    used4_prefix : str
+        Flow of 4.
+
+    time4_prefix : str
+        Estimated depletion time of 4.
+
+    start5_prefix : str
+        Starting volume of 5.
+
+    used5_prefix : str
+        Flow of 5.
+
+    time5_prefix : str
+        Estimated depletion time of 5.
+
+    start6_prefix : str
+        Starting volume of 6.
+
+    used6_prefix : str
+        Flow of 6.
+
+    time6_prefix : str
+        Estimated depletion time of 6.
+
+    start7_prefix : str
+        Starting volume of 7.
+
+    used7_prefix : str
+        Flow of 7.
+
+    time7_prefix : str
+        Estimated depletion time of 7.
+
+    start8_prefix : str
+        Starting volume of 8.
+
+    used8_prefix : str
+        Flow of 8.
+
+    time8_prefix : str
+        Estimated depletion time of 8.
+
+    start9_prefix : str
+        Starting volume of 9.
+
+    used9_prefix : str
+        Flow of 9.
+
+    time9_prefix : str
+        Estimated depletion time of 9.
+
+    start10_prefix : str
+        Starting volume of 10.
+
+    used10_prefix : str
+        Flow of 10.
+
+    time10_prefix : str
+        Estimated depletion time of 10.
+    """
+
+    integrator_source = UCpt(EpicsSignal)
+    flow_source = UCpt(EpicsSignal)
+    names = UCpt(EpicsSignal)
+
+    start1 = UCpt(EpicsSignal)
+    used1 = UCpt(EpicsSignal)
+    time1 = UCpt(EpicsSignal)
+
+    start2 = UCpt(EpicsSignal)
+    used2 = UCpt(EpicsSignal)
+    time2 = UCpt(EpicsSignal)
+
+    start3 = UCpt(EpicsSignal)
+    used3 = UCpt(EpicsSignal)
+    time3 = UCpt(EpicsSignal)
+
+    start4 = UCpt(EpicsSignal)
+    used4 = UCpt(EpicsSignal)
+    time4 = UCpt(EpicsSignal)
+
+    start5 = UCpt(EpicsSignal)
+    used5 = UCpt(EpicsSignal)
+    time5 = UCpt(EpicsSignal)
+
+    start6 = UCpt(EpicsSignal)
+    used6 = UCpt(EpicsSignal)
+    time6 = UCpt(EpicsSignal)
+
+    start7 = UCpt(EpicsSignal)
+    used7 = UCpt(EpicsSignal)
+    time7 = UCpt(EpicsSignal)
+
+    start8 = UCpt(EpicsSignal)
+    used8 = UCpt(EpicsSignal)
+    time8 = UCpt(EpicsSignal)
+
+    start9 = UCpt(EpicsSignal)
+    used9 = UCpt(EpicsSignal)
+    time9 = UCpt(EpicsSignal)
+
+    start10 = UCpt(EpicsSignal)
+    used10 = UCpt(EpicsSignal)
+    time10 = UCpt(EpicsSignal)
```

Comment:
I'm sure there is a reason it has to look this way but its still upsetting

Reply:
https://i.gifer.com/Uh8z.gif

Reply:
Yeah, at least for now :/
If somebody ends up standardizing the PVs for the flow integrators, then this can be changed.

Reply:
Cleaner now